docs/polkadot/implementors-guide/guide.md

```diff
@@ -0,0 +1,544 @@
+
+The Approval Process looks like this:
+1. Parablocks that have been included by the Inclusion Pipeline are pending approval for a time-window known as the secondary checking window.
+1. During the secondary-checking window, validators randomly self-select to perform secondary checks on the parablock.
+1. These validators, known in this context as secondary checkers, acquire the parablock and its PoV, and re-run the validation function.
+1. The secondary checkers submit the result of their checks to the relay chain. Contradictory results lead to escalation, where even more secondary checkers are selected and the secondary-checking window is extended.
+1. At the end of the Approval Process, the parablock is either Approved or it is rejected. More on the rejection process later.
+
+These two pipelines sum up the sequence of events necessary to extend and acquire full security on a Parablock. Note that the Inclusion Pipeline must conclude for a specific parachain before a new block can be accepted on that parachain. After inclusion, the Approval Process kicks off, and can be running for many parachain blocks at once.
+
+[TODO Diagram: Inclusion Pipeline & Approval Processes interaction]
+
+It is also important to take note of the fact that the relay-chain is extended by BABE, which is a forkful algorithm. That means that different block authors can be chosen at the same time, and may not be building on the same block parent. Furthermore, the set of validators is not fixed, nor is the set of parachains. And even with the same set of validators and parachains, the validators' assignments to parachains is flexible. This means that the architecture proposed in the next chapters must deal with the variability and multiplicity of the network state.
+
+[TODO Diagram: Forkfulness]
+
+----
+
+## Architecture
+
+Our Parachain Host is a blockchain. A blockchain is a Directed Acyclic Graph (DAG) of state transitions, where every block can be considered to be the head of a linked-list (known as a "chain" or "fork") with a cumulative state which is determined by applying the state transition of each block in turn. All paths through the DAG terminate at the Genesis Block.
```

Comment:
"A blockchain is a Directed Acyclic Graph (DAG) of state transitions"

That is correct, of course, but a blockchain is also a tree, a more specific subset of DAGs. I think calling it a DAG here will give the impression that fork merging is possible which is not the case for the parachain host.

Reply:
The statement that each block is the head of a linked-list should technically does this already, but I will clarify.